app/src/main/java/com/android/doctorapp/ui/appointment/BookAppointmentFragment.kt

```diff
@@ -114,12 +128,34 @@ class BookAppointmentFragment :
         }
     }
 
+    private fun setDateAndTime(
```

Comment:
Remove this setDateAndTime method i think unused method.

Reply:
Done